src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/repo/RepoFactory.java

```diff
@@ -73,8 +76,22 @@ public RepoFactory(RepoConfig config, Database database) throws IOException {
 
   /** Lookup a repo by its repoId. */
   public Repo getInstance(String repoId) throws RepoStorageException, RepoNotFoundException {
+    return getInstance(repoId, false);
+  }
+
+  public Repo getInstance(String repoId, boolean includeCreating)
```

Comment:
Could you use a two value enum for includeCreating rather than a boolean - it makes the code more readable at the callsite?

Reply:
I've split into two calls instead, as almost every caller uses the `false` version above.